src/form.svelte

```diff
@@ -42,36 +56,53 @@
       return;
     }
 
-    titlePlaceholder = tabMetadata.metadata.title;
-    descriptionPlaceholder = tabMetadata.metadata.description;
+    titlePlaceholder = tabMetadata.metadata.title ?? "";
+    descriptionPlaceholder = tabMetadata.metadata.description ?? "";
 
     const existingBookmark = tabMetadata.bookmark;
     if (existingBookmark) {
-      bookmarkExists = true;
+      bookmarkId = existingBookmark.id;
       title = existingBookmark.title;
-      tags = existingBookmark.tag_names ? existingBookmark.tag_names.join(" ") : "";
+      tags = existingBookmark.tag_names
+        ? existingBookmark.tag_names.join(" ")
+        : "";
       description = existingBookmark.description;
       notes = existingBookmark.notes;
       unread = existingBookmark.unread;
     }
   }
 
   async function handleSubmit() {
-    const tagNames = tags.split(" ").map(tag => tag.trim()).filter(tag => !!tag);
+    const tagNames = tags
+      .split(" ")
+      .map((tag) => tag.trim())
+      .filter((tag) => !!tag);
+
     const bookmark = {
       url,
-      title,
-      description,
+      title: title || titlePlaceholder,
+      description: description || descriptionPlaceholder,
```

Comment:
I think this may be a bug in the current extension, but in any case this change also works as expected.